{{ cookiecutter.__project_name_kebab }}/ruff.toml

```diff
@@ -0,0 +1,30 @@
+extend-exclude = [
+    "migrations",
+    "LC_MESSAGES",
+    "locale",
+]
```

Comment:
most of the files we were previously ignoring are in ruff's base config anyway so I've just extended with the handful that weren't

Reply:
q: why not lint/format migrations?

Reply:
I was trying to match the original configuration we had for flake8 and isort, so it is copied from

https://github.com/wagtail/cookiecutter-wagtail-package/blob/734ec2ef705b8b2d6ffa1bed46c0c6f8e8d65dd2/%7B%7B%20cookiecutter.__project_name_kebab%20%7D%7D/setup.cfg#L3
and
https://github.com/wagtail/cookiecutter-wagtail-package/blob/734ec2ef705b8b2d6ffa1bed46c0c6f8e8d65dd2/%7B%7B%20cookiecutter.__project_name_kebab%20%7D%7D/setup.cfg#L10

In general, I'm on mostly on-board with opting machine-generated files out of code style. But then, humans write migrations too.

I've removed it in 6abb0172